frontend/app/src/components/display/PercentageDisplay.vue

```diff
@@ -26,12 +26,10 @@ const props = defineProps({
 });
 
 const { assetPadding, value } = toRefs(props);
-const { shouldShowPercentage, scrambleData } = storeToRefs(
-  useSessionSettingsStore()
-);
+const { shouldShowPercentage } = storeToRefs(useSessionSettingsStore());
 
 const displayValue = computed<string>(() => {
-  if (get(scrambleData) || !get(shouldShowPercentage)) {
+  if (!get(shouldShowPercentage)) {
```

Comment:
can we also use the multiplier for the percentages?

Reply:
hmm should we? since the value is multiplied by same constant, the percentage shouldn't really matter (should be the same). Plus if we scramble it, the value will not add up.